pyproject.toml

```diff
@@ -9,6 +9,7 @@ exclude = '''
     | \.venv
     | htmlcov
     | venv
+    | lib
```

Comment:
Could you add a comment on why this is ignored, it's not obvious to me as someone unfamiliar with the project!